src/toolboxes/paintbrush/UI.tsx

```diff
@@ -1,32 +1,16 @@
 import React, { ChangeEvent, ReactElement } from "react";
-import {
-  Tooltip,
-  IconButton,
-  Accordion,
-  AccordionSummary,
-  Typography,
-  AccordionDetails,
-  Grid,
-} from "@material-ui/core";
-
-import {
-  ExpandMore,
-  Brush,
-  RadioButtonUncheckedSharp,
-} from "@material-ui/icons";
-
-import { Tool } from "@/tools";
+import { Popover } from "@material-ui/core";
 
 import { BaseSlider } from "@/components/BaseSlider";
 import { Sliders, SLIDER_CONFIG } from "./configSlider";
 
 import { usePaintbrushStore } from "./Store";
 
 interface Props {
-  expanded: boolean;
-  activeTool: Tool;
-  onChange: (event: ChangeEvent, isExpanded: boolean) => void;
-  activateTool: (tool: Tool) => void;
+  open: boolean;
+  anchorEl: HTMLButtonElement | null;
```

Comment:
What is El?

Reply:
A HTML element, or a function that returns it. It's used to set the position of the popover menu. 

https://material-ui.com/api/menu/#props

Reply:
Can we use the word element then? Rather than el?